qiskit_bot/notifications.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+import io
+import logging
+import multiprocessing
+import os
+import re
+
+import fasteners
+
+from qiskit_bot import git
+
+LOG = logging.getLogger(__name__)
+
+with io.StringIO() as buf:
+    buf.write("Thank you for opening a new pull request.\n\n")
+    buf.write(
+        "Before your PR can be merged it will first need to pass "
+        "continuous integration tests and be reviewed. Sometimes "
+        "the review process can be slow, so please be patient.\n\n"
+    )
+    buf.write(
+        "While you're waiting, please feel free to review other "
+        "open PRs. While only a subset of people are authorized "
+        "to approve pull requests for merging, everyone is "
+        "encouraged to review open pull requests. Doing reviews "
+        "helps reduce the burden on the core team and helps make "
+        "the project's code better for everyone.\n"
+    )
+    DEFAULT_PRELUDE = buf.getvalue()
```

Comment:
Again, not important at all: is there an advantage to defining this here rather than as part of the schema/configuration file?

(I'm guessing testing?)

Reply:
Yeah, it's mostly testing I think, but I honestly wrote this long enough ago I'm not really sure